### ISSUE TITLE
Parametrize subtest

### ISSUE THREAD

**Issue by jrieke** [pytest-dev/pytest-subtests#36]
Is it in any way possible to parametrize a subtest with `@pytest.mark.parametrize`? I'm dynamically creating subtests for different templates that I need to check but it would be nice if I could insert values into those templates via parametrize. (Of course, I can also just pass them in as normal values, but pytests formatting with parametrize is nicer/easier to understand and debug).

**Comment by RonnyPfannschmidt** [comment]
no, subtests happen after parametrize - but you can pass the parameter values to subtest to make them known
